storage/rocksdb/rdb_datadic.h

```diff
@@ -379,31 +390,31 @@ class Rdb_key_def
     SECONDARY_FORMAT_VERSION_LATEST= SECONDARY_FORMAT_VERSION_UPDATE1,
   };
 
-  void setup(const TABLE *table, const Rdb_tbl_def *tbl_def);
+  void setup(const TABLE* const table, const Rdb_tbl_def* const tbl_def);
 
   rocksdb::ColumnFamilyHandle *get_cf() const { return m_cf_handle; }
 
   /* Check if keypart #kp can be unpacked from index tuple */
-  inline bool can_unpack(uint kp) const;
+  inline bool can_unpack(const uint &kp) const;
```

Comment:
Changes like this look suspicious - why would we want to pass an unsigned int by a const reference instead of by value? This adds dereference cost